content/hi/docs/reference/glossary/affinity.md

```diff
@@ -0,0 +1,22 @@
+---
+title: आत्मीयता
+id: affinity
+date: 2019-01-11
+full_link: /docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
+short_description: >
+     पॉड्स को कहां रखा जाए, यह निर्धारित करने के लिए शेड्यूलर द्वारा उपयोग किए जाने वाले नियम
+aka:
+tags:
+- fundamental
+---
+
+कुबेरनेट्स में, _affinity_ नियमों का एक समूह है जो शेड्यूलर को संकेत देता है कि पॉड्स को कहाँ रखा जाए।
```

Comment:
```suggestion
कुबेरनेट्स में, _आत्मीयता_ नियमों का एक समूह है जो शेड्यूलर को संकेत देता है कि पॉड्स को कहाँ रखा जाए।
```

Reply:
Hey @divya-mohan0209, I think it is a keyword and should remain in english. I guess other reference glossary also follow this.

Reply:
In the entire doc, you've translated it to आत्मीयता except this one place and if we're retaining the keyword, it needs to be written in devnagari script.

For reference though, I did check the Korean localization & it didn't seem like this was the case. Link to the Korean translation for reference: https://github.com/kubernetes/website/blob/main/content/ko/docs/reference/glossary/affinity.md

Reply:
Thanks @divya-mohan0209, I have made the changes